db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160128204311) do
+ActiveRecord::Schema.define(version: 20200411183221) do
```

Comment:
OMG the schema version hasn't been updated since Jan 2016 😱

Reply:
omg